### ISSUE TITLE
Trained Model

### ISSUE THREAD

**Issue by HarbingerOfHell** [GauthierDmn/question_generation#2]
Hi,
Due to the large time taken to train the model, is there any way that you could provide your model?

**Comment by GauthierDmn** [comment]
Hi @HarbingerOfHell,

I don't have model weights to share at the moment. Now if you train your model on Google Collab notebooks using the free GPU, you will be able to have decent results in only a few hours.

Gauthier

**Comment by sreejith3534** [comment]
yes it is taking too much time to even start the first epoch even in GPU
